test/integration/selenium-helpers.js

```diff
@@ -17,6 +19,105 @@ const {By, Key, until} = webdriver;
 // The Jasmine default timeout is 30 seconds so make sure this is lower.
 const DEFAULT_TIMEOUT_MILLISECONDS = 20 * 1000;
 
+// This removes confusing `regenerator-runtime` noise from stack traces.
+// This is V8-specific code. Please don't use it in a browser or any production code.
+const oldPrepareStackTrace = Error.prepareStackTrace;
+Error.prepareStackTrace = function (error, stack) {
+    stack = stack.filter(callsite => {
+        const filename = callsite.getFileName();
+        return filename && !filename.includes('regenerator-runtime');
+    });
+    if (oldPrepareStackTrace) {
+        return oldPrepareStackTrace(error, stack);
+    }
+    return [
+        `${error.constructor.name}: ${error.message}`,
+        ...stack.map(callsite => `    at ${callsite.toString()}`)
+    ].join('\n');
+};
+
+/**
+ * An error thrown by SeleniumHelper.
+ * @extends Error
+ */
+class SeleniumHelperError extends Error {
```

Comment:
This is a bit extra, but it does make troubleshooting easier. Because Jest uses `regenerator-runtime` to re-implement promises, the stack reported by an exception thrown in an async function isn't what you might expect. The filter above hides any lines that talk about `regenerator-runtime`, but adding in lines that mention, say, the test that's failing (!!) requires jumping through extra hoops.

Before these changes, test failure output was typically something like this -- without any indication of which test was running at the time of the exception.
```
Error: Timeout - Async callback was not invoked within the 60000ms timeout specified by jest.setTimeout.
    at mapper (/home/circleci/repo/node_modules/jest-jasmine2/build/queue_runner.js:41:52)
    at /home/circleci/repo/node_modules/jest-jasmine2/build/queue_runner.js:87:41
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
StaleElementReferenceError: stale element reference: stale element not found
  (Session info: chrome=117.0.5938.62)
  (Driver info: chromedriver=117.0.5938.62 (25a7172909a4cba7355365cf424d7d7eb35231f4-refs/branch-heads/5938@{#1146}),platform=Mac OS X 10.15.4 x86_64)
    at Object.checkLegacyResponse (/home/circleci/repo/node_modules/selenium-webdriver/lib/error.js:575:15)
    at parseHttpResponse (/home/circleci/repo/node_modules/selenium-webdriver/lib/http.js:639:13)
    at Executor.execute (/home/circleci/repo/node_modules/selenium-webdriver/lib/http.js:573:28)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at thenableWebDriverProxy.execute (/home/circleci/repo/node_modules/selenium-webdriver/lib/webdriver.js:735:17)
```

After these changes, output is more like this, and even better when run with `chromedriver` (locally) instead of Sauce Labs (though CI):
```
SeleniumHelperError: clickText failed
    text: Raw
clickXpath failed
    xpath: //*[contains(text(), 'Raw')]
element not interactable
  (Session info: chrome=117.0.5938.62)
  (Driver info: chromedriver=117.0.5938.62 (25a7172909a4cba7355365cf424d7d7eb35231f4-refs/branch-heads/5938@{#1146}),platform=Mac OS X 10.15.4 x86_64)
(no webdriver context collected)
    at SeleniumHelper._callee5$ (/home/circleci/repo/test/integration/selenium-helpers.js:345:28)
    at step (/home/circleci/repo/test/integration/selenium-helpers.js:7:191)
    at /home/circleci/repo/test/integration/selenium-helpers.js:7:437
    at SeleniumHelper.<anonymous> (/home/circleci/repo/test/integration/selenium-helpers.js:7:99)
    at SeleniumHelper.clickText (/home/circleci/repo/test/integration/selenium-helpers.js:560:30)
    at wrapper (/home/circleci/repo/node_modules/lodash.bindall/index.js:550:15)
    at clickText (/home/circleci/repo/test/integration/project-page.test.js:202:19)
    at step (/home/circleci/repo/test/integration/project-page.test.js:9:191)
    at /home/circleci/repo/test/integration/project-page.test.js:9:361
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```